keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,587 @@
+
+
+def round_repeats(repeats, depth_coefficient):
+    """Round number of repeats based on depth multiplier."""
+    return int(math.ceil(depth_coefficient * repeats))
+
+
+def get_conv_constructor(conv_type):
+    if conv_type == "mb_conv":
+        return MBConvBlock
+    elif conv_type == "fused_mb_conv":
+        return FusedMBConvBlock
+    else:
+        raise ValueError(
+            "Expected `conv_type` to be "
+            "one of 'mb_conv', 'fused_mb_conv', but got "
+            f"`conv_type={conv_type}`"
+        )
+
+
+def get_block_conv(
+    conv_type,
+    input_filters,
+    output_filters,
+    expand_ratio,
+    kernel_size,
+    strides,
+    squeeze_and_excite_ratio,
+    activation,
+    survival_probability,
+    name,
+):
+    conv_constructor = get_conv_constructor(conv_type)
+
+    return conv_constructor(
+        input_filters=input_filters,
+        output_filters=output_filters,
+        expand_ratio=expand_ratio,
+        kernel_size=kernel_size,
+        strides=strides,
+        se_ratio=squeeze_and_excite_ratio,
+        activation=activation,
+        bn_momentum=0.9,
+        survival_probability=survival_probability,
+        name=name,
+    )
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class EfficientNetV2Backbone(Backbone):
+    """Instantiates the EfficientNetV2 architecture.
+
+    Args:
+        include_rescaling: bool, whether to rescale the inputs. If set
+            to `True`, inputs will be passed through a `Rescaling(1/255.0)`
+            layer.
+        width_coefficient: float, scaling coefficient for network width.
+        depth_coefficient: float, scaling coefficient for network depth.
+        default_size: integer, default input image size.
+        stackwise_kernel_sizes:  list of ints, the kernel sizes used for each
+            conv block.
+        stackwise_num_repeats: list of ints, number of times to repeat each
+            conv block.
+        stackwise_input_filters: list of ints, number of input filters for
+            each conv block.
+        stackwise_output_filters: list of ints, number of output filters for
+            each stack in the conv blocks model.
+        stackwise_expand_ratios: list of floats, expand ratio passed to the
+            squeeze and excitation blocks.
+        stackwise_squeeze_and_excite_ratios: list of ints, the squeeze and
+            excite ratios passed to the squeeze and excitation blocks.
+        stackwise_strides: list of ints, stackwise_strides for each conv block.
+        stackwise_conv_types: list of strings.  Each value is either 'mb_conv'
+            or 'fused_mb_conv' depending on the desired blocks.
+        dropout_rate: float, dropout rate before final classifier layer.
+        drop_connect_rate: float, dropout rate at skip connections.
+        depth_divisor: integer, a unit of network width.
+        min_depth: integer, minimum number of filters.
+        activation: activation function.
```

Comment:
Activation for what? Let's add some color.

Reply:
Yep yep - added